packages/otetris/otetris.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "lord@crocodile.org"
+authors: "Vadim Zaliva"
+homepage: "https://github.com/vzaliva/otetris"
+bug-reports: "https://github.com/vzaliva/otetris/issues"
+license: "MIT"
+dev-repo: "https://github.com/vzaliva/otetris.git"
+build: ["jbuilder" "build"]
+install: ["jbuilder" "install"]
+remove: [
```

Comment:
as you are using `jbuilder` you don't need to specify the `remove` command: this is automatically done by using an `.install` file when the package is installed.